parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/ColumnChunkMetaData.java

```diff
@@ -77,6 +80,22 @@ public static ColumnChunkMetaData get(
       long valueCount,
       long totalSize,
       long totalUncompressedSize) {
+    return get(path, new PrimitiveType(Type.Repetition.OPTIONAL, type, ""), codec, encodingStats, encodings, statistics,
```

Comment:
Use the `Types` API instead of constructors. The constructors will probably not be public in the 2.0 API.